static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -40,10 +41,10 @@ Once initialized in a project, DVC populates its installation directory
 
 ## Structure of cache directory
 
-The structure of directory depends if the data is stored in a file or is in a
-directory.
+There are two ways in which the data is stored in cache which depends on if the
```

Comment:
`which ... which`  - very hard to read. Let's simplify something like - `There are two ways in which the data is stored in cache. It depends ...`